packages/fluentui/react-northstar-prototypes/src/prototypes/TreeAdditionalActionsPrototype/components/TreeActionsModifyAriaDisable.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { treeAsListboxBehavior, Box, Tree, Button, Flex, Text } from '@fluentui/react-northstar';
+
+const TreeActionsUsingPopup = ({ useMacAccessabilityBehavior }) => {
+  const items = [
+    {
+      id: 'tree-item-0',
+      title: 'House Targaryen',
+      items: [
+        {
+          id: 'tree-item-01',
+          title: 'Aerys',
+          items: [
+            {
+              id: 'tree-item-011',
+              title: 'Rhaegar',
+            },
+            {
+              id: 'tree-item-012',
+              title: 'Viserys',
+            },
+            {
+              id: 'tree-item-013',
+              title: 'Daenerys',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      id: 'tree-item-1',
+      title: {
+        children: (C, p) => {
+          return <TreeTitleModifyAriaDisableButton />;
```

Comment:
Shouldn't `Component` be used here to keep `TreeItem` behaviors and styles? I think this would probably break selectable item for example?